### DISCUSSION TITLE
Restrict acess to my terrain

### DISCUSSION THREAD

**Discussion by leomanjos** [GriefPrevention/GriefPrevention#2166] [Q&A]
It's possible to restrict acess of someone on my terrain?

**Comment by bigpresh** [comment]
Well, you can, by building a secure structure they can't get inside of, but there's no way with GP itself to set it so that people can't walk into your claim other than physical blockers like walls.  The ability to prevent people entering a claim was discussed in the past and rejected as the movement checks to implement it would become expensive resource-wise (each time any player moves, their new location would have to be checked against all claims, which doesn't sound very scalable).

I believe the GPFlags addon implements this though - have a look at the `NoEnter` flag on https://github.com/akdukaan/GPFlags/wiki/Flags
